src/Table/TableHead.js

```diff
@@ -92,22 +92,22 @@ class TableHead extends Component {
         >
           <div
             className={classNames(
-                {
-                  [theme[`${align}Align`]]: !orderable,
-                },
-                theme.tableHeadItem
-              )}
+              {
+                [theme[`${align}Align`]]: !orderable,
+              },
+              theme.tableHeadItem
+            )}
           >
             {title}
           </div>
         </th>
       )
     }
 
-    const thProps = disabled || not(onOrderChange) ? {} :
-      {
-        onClick: () => this.handleOrderChange(index),
-      }
+    const thProps = disabled
```

Comment:
```suggestion
    const thProps = disabled || not(onOrderChange)
```
maybe is better to read if there is no line break on conditional